docs/server/source/http-client-server-api.rst

```diff
@@ -161,6 +165,25 @@ Transactions
    :statuscode 202: The pushed transaction was accepted in the ``BACKLOG``, but the processing has not been completed.
    :statuscode 400: The transaction was malformed and not accepted in the ``BACKLOG``.
 
+.. http:post:: /api/v1/transactions?mode={mode}
+
+   .. note::
+
+       This option is only available when using BigchainDB with Tendermint.
+
+   Tendermint offers a `broadcast API
+   <http://tendermint.readthedocs.io/projects/tools/en/master/using-tendermint.html#broadcast-api>`_ with three different modes to send transactions.
+   By setting the mode, a new transaction can be pushed with a different mode than the default. The default mode is ``broadcast_tx_async``, which
+   will return immediately and not wait to see if the transaction is valid.
+
+   :query string mode: (Optional) One of the three supported modes to send a transaction: ``broadcast_tx_async``, ``broadcast_tx_sync``, ``broadcast_tx_commit``.
+
+   The only change for a request, compared to the one above, is to set the mode
+
+    .. sourcecode:: http
+
+        GET /api/v1/transactions?mode={mode} HTTP/1.1
```

Comment:
Maybe make this an example rather than this generic line, which is the same as the subsection heading, e.g.

...is to set the mode. For example:

    GET /api/v1/transactions?mode=commit HTTP/1.1

Reply:
Actually, are the expected response status codes the same, i.e. `202 Accepted` and `400 Bad Request`?

Reply:
The response status codes don't change.

Reply:
ok